lib/intersection.h

```diff
@@ -61,8 +60,7 @@ inline float intersect_ray_plane(const Ray& ray, const Vec& v0, const Vec& n) {
  *   true, if the ray intersects the triangle, otherwise false
  */
 inline bool intersect_ray_triangle(const Ray& ray, const Triangle& tri,
```

Comment:
How wide is this line? I'm looking at the diff on my 13'' Mac with split view. Ideally the view does not break lines. That's an argument for 80 character width.